flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/ShowCreateUtil.java

```diff
@@ -177,6 +206,17 @@ static String extractFormattedColumns(ResolvedCatalogBaseTable<?> table, String
                 .collect(Collectors.joining(",\n"));
     }
 
+    static Optional<String> extractFormattedColumns(ResolvedSchema schema) {
```

Comment:
do we need printIndent here? cc line 203.

Reply:
This is formatted differently from table. In table, each column is a row (joined by `\n'). In model, columns for `input` and `output` are in same row respectively (joined by `,`). I feel it's ok since the input/output columns for model won't be too many. wdyt?